src/plugins/data/common/search/aggs/buckets/date_histogram.ts

```diff
@@ -10,7 +10,13 @@ import { get, noop, find, every } from 'lodash';
 import moment from 'moment-timezone';
 import { i18n } from '@kbn/i18n';
 
-import { KBN_FIELD_TYPES, TimeRange, TimeRangeBounds, UI_SETTINGS } from '../../../../common';
+import {
+  IndexPatternField,
+  KBN_FIELD_TYPES,
+  TimeRange,
+  TimeRangeBounds,
+  UI_SETTINGS,
+} from '../../../../common';
```

Comment:
nit: I don't like when we do imports from the parent index file. I understand it's not your code, but could you please import it directly from
`import { IFieldType } from '../../../index_patterns'`